core/src/main/java/org/eclipse/krazo/binding/convert/ConverterRegistry.java

```diff
@@ -37,24 +36,12 @@
 
     @PostConstruct
     public void init() {
-        register(new ShortConverter());
-        register(new IntegerConverter());
-        register(new LongConverter());
-        register(new DoubleConverter());
-        register(new FloatConverter());
-        register(new BigDecimalConverter());
-        register(new BigIntegerConverter());
-        register(new BooleanConverter());
+        converters.addAll(ServiceLoaders.list(MvcConverter.class));
     }
 
-    private void register(MvcConverter converter) {
-        converters.add(converter);
-    }
-
-    <T> MvcConverter<T> lookup(Class<T> rawType, Annotation[] annotations) {
+    public <T> MvcConverter<T> lookup(Class<T> rawType) {
```

Comment:
I wonder if we should keep the annotations and also add them to `MvcConverter#supports()` and maybe even `MvcConverter#convert()`. Actually it was mistake that I didn't add it in the first version. I could imagine that this would be useful if you want to customize the conversion for a standard data type only for certain scenarios. Like for example if you want to add an annotation like `@ForceWireFormat`. Thoughts?

Reply:
Sure, we can re-add it. I didn't saw any usage at the moment, so I removed it. But maybe we should re-add it in a separate PR, so this one doesn't contain too much new features.

Reply:
Sure, I agree, let's do it in a subsequent pull request.